articles/custom-domains/additional-configuration.md

```diff
@@ -0,0 +1,115 @@
+});
+```
+
+Note that the Management API only accepts Auth0 domains. If you use a custom domain and also intend to perform [Management API actions with Auth0.js](/libraries/auth0js/v9#user-management), you must instantiate a new copy of `webAuth` using your Auth0 domain.
+
+## Use custom domains in emails
+
+If you want to use your custom domain with your Auth0 emails, you must enable this feature. 
+
+Go to [Dashboard > Tenant Settings > Custom Domains](${manage_url}/#/tenant/custom_domains) and enable the **Use Custom Domain in Emails** toggle. When the toggle is green, this feature is enabled.
+
+![Use Custom Domain in Emails toggle](/media/articles/custom-domains/cd_email_toggle.png)
+
+## Configure social identity providers
+
+If you want to use social identity providers with your custom domain, you must update the [Allowed Callback URLs](${manage_url}/#/clients/${account.clientId}/settings) to include your custom domain (such as `https://login.northwind.com/login/callback`).
+
+::: warning
+You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
+:::
+
+## Configure SAML identity providers
+
+If you want to use SAML identity providers (IdPs) with your custom domain, you must get the service provider metadata from Auth0 (such as `https://YOUR-CUSTOM-DOMAIN/samlp/metadata?connection=YOUR-CONNECTION-NAME`). This includes updated **Assertion Consumer Service (ACS) URLs**. Then, you have to manually update this value in your IdP(s). This change to your IdP(s) must happen at the same time as you begin using your custom domain in your applications. This can pose a problem if there are multiple IdPs to configure.
+
+Alternatively, you can use signed requests to fulfill this requirement:
+- Once your custom domain is set up, go to [Dashboard > Tenant Settings > Custom Domains](${manage_url}/#/tenant/custom_domains) and download the certificate from the link under the **Sign Request** toggle
+- Give the certificate to the IdP(s) to upload. This enables the IdP to validate the signature on the `AuthnRequest` message that Auth0 sends to the IdP
+- The IdP will import the certificate and if necessary, signature verification should be enabled (exact steps vary by IdP)
+- Turn on the **Sign Request** toggle in the Dashboard under **Connections > Enterprise > SAMLP > CONNECTION**. This will trigger Auth0 to sign the SAML `AuthnRequest` messages it sends to the IdP
+
+Once this is done, and you start using your custom domain, the IdP will receive that custom domain in your signed request. Because your application is trusted, the IdP should automatically override whatever was configured as your ACS URL and replace it with the value sent in the signed request. This will prevent you from having to change one or many IdP settings all at the same time, instead allowing you to prepare them to accept your signed requests ahead of time. you can then at a later date have the IdPs change the ACS URL as well.
```

Comment:
We should add here that not all IDPs accept the ACS URL in the signed request. So they should check with their IDP to confirm it's supported or not.